src/adaXT/predict/predict.pyi

```diff
@@ -20,8 +20,16 @@ class PredictRegression(Predict):
 
     pass
 
-class PredictLinearRegression(Predict):
+class PredictLocalPolynomial(Predict):
```

Comment:
From what I can tell this is not the default in the base_model.pyx

Reply:
Double checked but seems to be correct